lib/project.ml

```diff
@@ -4,20 +4,47 @@ type t = Fpath.t
 
 let folder_blacklist = [ "_build"; "_opam"; Fpath.to_string Config.vendor_dir ]
 
+let repo_filename = "repo"
+
+let packages_folder = "packages"
```

Comment:
According to opam devs this folder doesn't need to be called `"packages"`, so a repo could potentially name this differently.

Reply:
I don't think it can, see: https://github.com/ocaml/opam/blob/master/src/repository/opamRepositoryPath.mli#L34

Reply:
It _might_ still be useful to use that function, in the case that they decide to change it in a future OPAM version (not that that's particularly likely though).

Reply:
Yes that was the plan, just pushed the fix!